src/styles/global.css

```diff
@@ -612,3 +612,58 @@ a.release-update::before {
   font-family: 'Glyphicons Regular';
   src: url(/fonts/glyphicons/glyphicons-regular-v192.woff2) format('woff2');
 }
+.addsearch-searchresults h3{
+	color: #000 !important;
```

Comment:
It's been many years since my regular CSS writing days, but I think `!important` is still only for use as a last resort. Can you make the selector more specific?